README.md

```diff
@@ -443,6 +443,7 @@ Metadata at execution time is structured as follows:
   noop: false, # A boolean indicating if you are running in noop mode. ruby_command blocks are never evaluated in noop mode
   auto: false, # A boolean indicating if you are running in auto mode
   paranoid: true, # A boolean indicating if you are running in paranoid mode (prompting before each step)
+  save_panes: false, # A boolean indicating if you do not want to close tmux panes ever
```

Comment:
Optional nitpick: if we change this to `keep_panes`, could. we say:

```
# A boolean indicating whether panes should be kept open after completion
```

@ClashTheBunny - do you think that makes sense?

Reply:
Love it!  Replaced!